introduction.tex

```diff
@@ -1,3 +1,15 @@
+\section{Abstract}
+France is in pursuit of closing the fuel cycle by transitioning from a \gls{LWR} fleet
+to a \gls{SFR} one with recycling of \gls{SNF}. Past transition scenarios looked at
+deploying additional \glspl{LWR} in order to meet \gls{SNF} demands for \gls{MOX} production.
+This paper uses CYCLUS to check the feasibility of using \gls{SNF}from other EU nations
+for French transition into a \gls{SFR}fleet without additional construction of \glspl{LWR}.
+A CYCLUS simulation is run from 1950 to 2160 for EU to track the \gls{SNF} mass
+and how much reprocessing and \gls{MOX} fabrication capacity is required to fuel
```

Comment:
"and how much reprocessing and \gls{MOX} fabrication capacity is required to fuel" -> "and to determine the necessary reprocessing and \gls{MOX} fabrication capacity to support"